kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/SyncSchedule/EditSyncSchedule.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+
+  <ImmersivePage
+    :appBarTitle="$tr('toolbarHeader')"
+  >
+    <KPageContainer>
+      <KGrid gutter="48">
+        <KGridItem>
+          <h1>{{ $tr('editSyncScheduleSubTitle') }}</h1>
+        </KGridItem>
+        <KGridItem>
+          <p>{{ $tr('deviceName') }}</p><br>
+        </KGridItem>
+
+        <KGridItem>
+          <KDropdownMenu
+            text="Every After 5 minutes"
```

Comment:
The menu options here should be translatable strings. I just refactored`KDropdownMenu` component, so we can use the new version. Let's plan a time to talk through that -- I'll show you the new documentation and we can talk about how to create a set of options for selection.